src/main/java/org/broadinstitute/hellbender/tools/copynumber/utils/annotatedregion/AnnotatedInterval.java

```diff
@@ -0,0 +1,102 @@
+package org.broadinstitute.hellbender.tools.copynumber.utils.annotatedregion;
+
+import com.google.common.collect.ImmutableSortedMap;
+import htsjdk.samtools.util.Locatable;
+import htsjdk.tribble.Feature;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+
+import java.util.SortedMap;
+
+/**
+ * Simple class that just has an interval and sorted name-value pairs.
+ */
+public final class AnnotatedInterval implements Locatable, Feature {
```

Comment:
The package name should be changed to reflect the name change to `AnnotatedInterval`.  You should probably also edit the commit message for this branch.

Also note https://github.com/broadinstitute/gatk/issues/3884.  I don't mind the redundancy for now.

Reply:
Renamed the package.

Done.